setup.py

```diff
@@ -7,7 +7,7 @@
     install_requires=[
         'astropy~=4.2',
         'ctapipe~=0.10.0',
-        'protozfits~=2.0.1',
+        'protozfits~=2.0.1.post1',
```

Comment:
No, this does not work. No you fix protozfits to `2.0.1` since the ~= operator only allows variation in the least significant version number you have included.

Since this release only fixes a build issue under specific cirmcumstances, I wouldn't even update the requirement.

Anyone installing it now will get the newest version anyway.

Reply:
ok, sorry, I did not know, then I'll make the new release without this and include it in lstchain